data.sql

```diff
@@ -1,6 +1,21 @@
-/* Populate database with sample data. */
+/* Populate database with sample data. (Milestone (1)) */
 
 insert into animals ( name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Agumon','feb 03, 2020', 0, true, 10.25);
 insert into animals ( name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Gabumon','nov 15, 2018', 2, true, 8);
 insert into animals ( name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Pikachu','Jan 7, 2021', 2, false, 15.04);
 insert into animals ( name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Devimon','May 12, 2017', 5, true, 11);
+
+/* (Milestone(2)) */
+ insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Charlander', 'feb 8, 2020', 0, false, 11);
+ insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Plantmon', 'nov 15, 2021', 2, true, 5);
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Squirtle', 'apr 2, 1993', 3, false, 12.13);
+ insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Angemon', 'jun 12, 2005', 1, true, 45);
+ insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Boarmon', 'jun 7, 2005', 7, true, 20.4);
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Blossom', 'oct 13, 1998', 3, true, 17);
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values ('Ditto', 'may 14, 2022', 4, true, 22);
```

Comment:
**It is great that you have managed your queries :blush:**

- [ ] Your first query has a name **Charlender** which is wrong. The right name is **Charmender**. Kindly fix it because the upcoming queries will be dependent on **Charmander** not **Charlender** :thinking:
- [ ] Some animals have weight in negative such as Charmander has **weight_kg = -11** but you have put **11** Kindly fix it for all your animals :blush: You can see the requirements in the image below :arrow_down:
![wrong-weights (copy)](https://user-images.githubusercontent.com/93039359/196954286-4039b123-915f-4b71-9115-eb6ddf507f3c.png)

Reply:
Ok mate thank You for putting in the effort ill work on it rn